docs/docusaurus/sidebars.json

```diff
@@ -35,15 +35,27 @@
       }
     ],
     "NMS User Guide": [
-      "nms/nms_overview",
-      "nms/nms_dashboard",
-      "nms/nms_equipment",
-      "nms/nms_network",
-      "nms/nms_subscriber",
-      "nms/nms_traffic",
-      "nms/nms_organizations",
-      "nms/nms_developer",
-      "nms/nms_how_to_guide"
+      {
+        "type": "subcategory",
+        "label": "NMS Usage",
+        "ids": [
+          "nms/nms_overview",
+          "nms/nms_dashboard",
+          "nms/nms_equipment",
+          "nms/nms_network",
+          "nms/nms_subscriber",
+          "nms/nms_traffic",
+          "nms/nms_organizations",
+          "nms/nms_how_to_guide"
+        ]
+      },
+      {
```

Comment:
Let's move this out of NMS user guide and call this NMS developer guide and add more pages under this.